src/app/editions/[id]/page.tsx

```
@@ -238,20 +285,52 @@ export default async function EditionDetailPage(props: Readonly<EditionDetailPag
+                                                                </h3>
+                                                                <span className="text-xs text-muted-foreground">
+                                                                    {teamAwards.length}
+                                                                </span>
+                                                            </div>
+
+                                                            <div className="space-y-3">
+                                                                {teamAwards.map((award, awardIndex) => {
+                                                                    const resourceUri = getAwardResourceUri(award);
+
+                                                                    return (
+                                                                        <AwardSection
```

ESLint

[error] 'AwardSection' is not defined.

(react/jsx-no-undef)